### PR TITLE
Propose ADR for Data Governance and Release Management for oscal-content

### PR DESCRIPTION
**Pull request by aj-stein-nist**
# Committer Notes

This decision relates to future management of oscal-content#116.

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- ~Do all automated CI/CD checks pass?~

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

(For reviewers: [The wiki has guidance](https://github.com/usnistgov/OSCAL/wiki/Issue-Review) on code review and overall issue review for completeness.)

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- ~Have you written new tests for your core changes, as applicable?~
- ~Have you included examples of how to use your new feature(s)?~
- ~Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.~

### CODE REVIEW

**Review by wendellpiez — DISMISSED** [review]
We very much need this or something like it.

Bravo AJ for bringing this forward. As you know I don't agree 100% with every point - all the more reason to agree to this in principle, so we can act on it, making any subsequent refinements meaningful. I look forward to conversations.

File: decisions/0008-oscal-content-management.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+### Data Management and Governance Challenges
+
+As enhancements and bug fixes for OSCAL increased, separate of the content, in between official upstream releases of SP 800-53A and SP 800-53B, staff and community members [identified bugs and enhancements to the OSCAL representations](). In most cases, these work items would not or did not diverge from the content in the official publication version. These data quality and OSCAL-specific enhancements would or did improve the ability of technical staff using the OSCAL representation to create or improve catalog and profile automation. There has been no clear guidance on how to accept these changes, publish them, and how to identify their versions upon release. These governance questions led to an accumulation of work items that delayed publication (at this time, read: merged into the `main` branch).
```
Comment (nikitawootten-nist) [inline decisions/0008-oscal-content-management.md:25]: B: What is the empty link (identified bugs and enhancements...) in the first sentence meant to go to?

**Review by nikitawootten-nist — CHANGES_REQUESTED** [review]
Overall very much approve. I understand now the release approach you were referring to.

Blocking comment below.

File: decisions/0008-oscal-content-management.md (hunk 1 above)
Comment (aj-stein-nist) [reply decisions/0008-oscal-content-management.md:25]: Woops! That's an easy one to fix. 🤦 

```suggestion
As enhancements and bug fixes for OSCAL increased, separate of the content, in between official upstream releases of SP 800-53A and SP 800-53B, staff and community members [identified bugs and enhancements to the OSCAL representations](https://github.com/usnistgov/oscal-content/issues). In most cases, these work items would not or did not diverge from the content in the official publication version. These data quality and OSCAL-specific enhancements would or did improve the ability of technical staff using the OSCAL representation to create or improve catalog and profile automation. There has been no clear guidance on how to accept these changes, publish them, and how to identify their versions upon release. These governance questions led to an accumulation of work items that delayed publication (at this time, read: merged into the `main` branch).
```

Comment (nikitawootten-nist) [reply decisions/0008-oscal-content-management.md]: Ah, ok fair enough!

**Review by nikitawootten-nist — DISMISSED** [review]

**Comment by aj-stein-nist** [comment]
I will merge this after usnistgov/oscal-content#204, since this will need to happen alongside that. Merging that PR means we are ready to roll. ⚡

**Review by iMichaela — DISMISSED** [review]

**Pull request merged**